docs/docusaurus/docs/guides/connecting_to_your_data/fluent/data_assets/how_to_organize_batches_in_a_sql_based_data_asset.md

```diff
@@ -82,47 +79,35 @@ For this guide, we will use a previously defined SQL Datasource named `"my_datas
 
 To retrieve this Datasource, we will supply the `get_datasource(...)` method of our Data Context with the name of the Datasource we wish to retrieve:
 
-```python title="Python code"
-my_datasource = context.get_datasource("my_datasource")
-my_asset = my_datasource.get_asset("my_asset")
+```python name="tests/integration/docusaurus/connecting_to_your_data/fluent_datasources/organize_batches_in_sqlite_datasource.py my_datasource"
 ```
 
 ### 3. Add a Splitter to the Data Asset
 
 Our table has a datetime column called "`pickup_datetime`" which we will use to split our TableAsset into Batches.
 
-```python title="Python code"
-table_asset.add_year_and_month_splitter(column_name="pickup_datetime")
+```python name="tests/integration/docusaurus/connecting_to_your_data/fluent_datasources/organize_batches_in_sqlite_datasource.py add_splitter_year_and_month"
 ```
 
 ### 4. (Optional) Add Batch Sorters to the Data Asset
 
 We will now add a Batch Sorter to our Data Asset.  This will allow us to explicitly state the order in which our Batches are returned when we request data from the Data Asset.  To do this, we will pass a list of sorters to the `add_sorters(...)` method of our Data Asset.
```

Comment:
This says, “add a Batch Sorter” but the next part mentions a list of sorter and the method name is ‘add_sorters’. 
```suggestion
We will now add Batch Sorters to our Data Asset.  This will allow us to explicitly state the order in which our Batches are returned when we request data from the Data Asset.  To do this, we will pass a list of sorters to the `add_sorters(...)` method of our Data Asset.
```